apps/emqx/src/emqx_ds_schema.erl

```diff
@@ -88,6 +88,14 @@ namespace() ->
 
 schema() ->
     [
+        {sessions,
+            ds_schema(#{
+                default =>
+                    #{
+                        <<"backend">> => ?DEFAULT_BACKEND
```

Comment:
I don't think we should make this backend configurable at all. For each type of release we should hardcode the backend offering the highest degree of consistency:
- OSS: local
- EE: raft
- ...